packages/manager/modules/octavia-load-balancer/src/create/controller.js

```diff
@@ -32,6 +37,59 @@ export default class OctaviaLoadBalancerCreateCtrl {
 
   onRegionChange(region) {
     this.model.region = region;
+    this.getPrivateNetworks();
```

Comment:
```suggestion
    return this.getPrivateNetworks();
```

Reply:
As getPrivateNetworks is not returning anything and only assign values into this. I don't think this is relevant